isis/tests/BundleResultsTests.cpp

```diff
@@ -0,0 +1,547 @@
+#include "BundleResults.h"
```

Comment:
I think that this file should be named UnitTestBundleResults.cpp according to the convention set by the gtest conversion project (https://github.com/USGS-Astrogeology/ISIS3/wiki/Writing-ISIS3-Tests-Using-Gtest-and-Ctest#test-names).